app/controllers/Checkout.scala

```diff
@@ -37,6 +36,14 @@ import scalaz.{NonEmptyList, OptionT}
 
 object Checkout extends Controller with LazyLogging with CatalogProvider {
 
+  val ukAndIsleOfMan = CountryGroup.UK.copy(countries = List(Country.UK, Country("IM", "Isle of Man")))
+  val weeklyUkNorthAmericaGroups = List (ukAndIsleOfMan, CountryGroup.US, CountryGroup.Canada)
+  val weeklyRowGroups = {
+    val rowUk = CountryGroup("Row Uk", "uk", None, CountryGroup.UK.countries.filterNot(ukAndIsleOfMan.countries.contains(_)), GBP, PostCode)
+    rowUk :: CountryGroup.allGroups.filterNot(group => (CountryGroup.UK :: weeklyUkNorthAmericaGroups) contains group)
+  }
```

Comment:
Are we able to deliver to all countries of all countrygroups or do we need a specific list of countries for Guardian Weekly ROW delivery?
This list currently include things like Antarctica